TombEngine/Scripting/Internal/TEN/Objects/ObjectIDs.h

```diff
@@ -1924,7 +1924,7 @@ static const std::unordered_map<std::string, GAME_OBJECT_ID> kObjIDs {
 	{ "SMOKE_EMITTER", ID_SMOKE_EMITTER },
 	{ "EARTHQUAKE", ID_EARTHQUAKE },
 	{ "BUBBLES", ID_BUBBLES },
-	{ "WATERFALLMIST", ID_WATERFALLMIST },
+	{ "WATERFALL_MIST", ID_WATERFALL_MIST },
```

Comment:
Object ID changes should be synchronous with changes in TE repo, otherwise nodes which operate on slot IDs may break.